app/src/main/res/values-is/strings.xml

```diff
@@ -0,0 +1,640 @@
+    <string name="customize">Sérsníða</string>
+    <string name="custom">Sérsniðið</string>
+    <string name="preselectedconfigs_title">Forvaldar uppsetningar</string>
+    <string name="orange">Appelsínu</string>
+    <string name="blue">Blátt</string>
+    <string name="green">Grænt</string>
+    <string name="password_no_match">Lykilorðin stemma ekki</string>
+    <string name="deselect_all">Afvelja allt</string>
+    <string name="filename">Skráarheiti</string>
+    <string name="compress_file_suggest_zip_extension">Mælt er með því að nota \".zip\" sem skráarendingu.</string>
+    <string name="about_amaze">Skráastjóri með Material-hönnun fyrir Android</string>
+    <string name="license">Notkunarleyfi</string>
+    <string name="libraries">Aðgerðasöfn</string>
+    <string name="password_repeat">Endurtaktu lykilorð</string>
+    <string name="create_file_suggest_txt_extension">Mælt er með því að nota \".txt\" sem skráarendingu.</string>
+    <string name="ssh_select_pem">Veldu einkalykil fyrir auðkenningu</string>
+    <string name="ssh_key_invalid_passphrase">Ógild aðgangssetning dulritunarlykils</string>
+    <string name="encryption_fail_copy">Mistókst að uppfæra dulritunarfærslu í gagnagrunni</string>
+    <string name="extracted">Afþjöppuð</string>
+    <string name="compressed">Þjöppuð</string>
+    <string name="processed">Meðhöndluð</string>
+    <string name="no_obtainable_info">Það eru engar upplýsingar um þessa skrá</string>
+    <string name="reopen_from_source">Tekst ekki, opnaðu aftur úr síðast notaða forriti</string>
+    <string name="no_file_error">Eitthvað fór úrskeiðis, hér er ekkert sem hægt er að opna</string>
+    <string name="file_read_only">Skráin sem var opnuð er aðeins til lestrar.</string>
+    <string name="got_it">Náði því!</string>
+    <string name="encrypt_file_save_as">Vista dulritaða skrá sem&#8230;</string>
+    <string name="encrypt_folder_save_as">Vista dulritaða möppu sem&#8230;</string>
+    <string name="encrypt_file_must_end_with_aze">Dulritaðar skrár verða að nota \".aze\" sem skráarendingu.</string>
+    <string name="cannot_overwrite">Get ekki yfirskrifað þessa skrá.</string>
+    <string name="saved_single_file">Skrá vistuð sem \'%s\'.</string>
+    <string name="saved_multi_files">%d skrár vistaðar.</string>
+    <string name="create_hidden_file_warn">Nýgerða skráin verður falin í skráalistanum.</string>
+    <string name="grant_apkinstall_permission">Forritið þarf heimild svo að uppsetning geti haldið áfram.</string>
+    <string name="time_remaining">Tími eftir</string>
+    <string name="transfer_rate">Flutningshraði</string>
+    <string name="unknown">óþekkt</string>
+    <string name="multiple_invalid_archive_entries">Ekki var hægt að afþjappa allar skrár í safnskránni; hún gæti verið skemmd eða af vafasömum uppruna.</string>
+    <string name="saf_otg_explanation">Af öryggisástæðum ferðu núna á skráakerfi Android. Veldu USB-tækið úr listanum til vinstri.</string>
+</resources>
```

Comment:
Please also translate `donation_thanks` too.

https://github.com/TeamAmaze/AmazeFileManager/blob/e5e5fe1b109ab5bda8a498cdc7524d83a47de1a2/app/src/main/res/values/strings.xml#L645

Reply:
Corrected the typo and added the missing string in a second commit (I'm not very used to git or github features, so pardon if I'm doing it wrong).